src/date.js

```diff
@@ -35,6 +35,24 @@ function validateRequiredCldr( path, value ) {
 }
 
 /**
+ * .loadIANA( json )
+ *
+ * @json [JSON]
+ *
+ * Load IANA data.
+ */
+Globalize.loadIANA = function( json ) {
```

Comment:
Pros:
- Letting users to load this data similarly to the way they do with CLDR is good. Therefore we don't embed such in the library.

Cons:
- Differently from CLDR, IANA doesn't provide with an official JSON binding. The one used here is non-official, it is using the JSON bindings from moment-timezone, and could be subject to changes. Ways to mitigate risks here could be 1) to get moment-timezone data put into a separate iana-json library; 2) check if IANA is interested in making it official? Ideas?